From ab30d1b8a94e2ba62f8844098ae0c2d9798a6d60 Mon Sep 17 00:00:00 2001
From: Alexandre Pereira da Silva <aletes.xgr@gmail.com>
Date: Fri, 5 Apr 2013 01:16:53 -0300
Subject: [PATCH 2/2] ARM: dts: imx23-olinuxino: Set the sdcard as
 non-removable

The imx23-olinuxino sdcard doesn't support card detect.

Signed-off-by: Alexandre Pereira da Silva <aletes.xgr@gmail.com>
---
 arch/arm/boot/dts/imx23-olinuxino.dts |    1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/imx23-olinuxino.dts b/arch/arm/boot/dts/imx23-olinuxino.dts
index e7484e4..03f55c2 100644
--- a/arch/arm/boot/dts/imx23-olinuxino.dts
+++ b/arch/arm/boot/dts/imx23-olinuxino.dts
@@ -29,6 +29,7 @@
 				pinctrl-names = "default";
 				pinctrl-0 = <&mmc0_4bit_pins_a &mmc0_pins_fixup>;
 				bus-width = <4>;
+				non-removable;
 				status = "okay";
 			};
 
-- 
1.7.10.4

